dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -646,4 +646,90 @@ def stub_country(code)
     assert_equal new_toolbox, options[:level]["toolbox"]
     assert_equal new_start, options[:level]["startBlocks"]
   end
+
+  test 'string_or_image can retrieve plain text' do
+    assert_equal string_or_image('match', "test"), "test"
```

Comment:
note that this is the only one of the added tests that is actually testing _new_ functionality; all the others are testing existing functionality